src/Integration.UnitTests/Binding/BindingProcessImplTests.cs

```diff
@@ -513,183 +410,6 @@ public void SilentSaveSolutionIfDirty()
             logger.AssertOutputStrings(0);
         }
 
-        [TestMethod]
-        public void DiscoverBindableProjects_AddsMatchingProjectsToBinding()
```

Comment:
I just deleted all of the `DiscoverBindableProject_XXX` tests since the method is now hard-coded to return `true`.
I'll remove it completely in the next PR.